<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DAPlotManageWidget</class>
 <widget class="QWidget" name="DAPlotManageWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>267</width>
    <height>601</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Chart Manage</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="spacing">
    <number>1</number>
   </property>
   <property name="leftMargin">
    <number>1</number>
   </property>
   <property name="topMargin">
    <number>1</number>
   </property>
   <property name="rightMargin">
    <number>1</number>
   </property>
   <property name="bottomMargin">
    <number>1</number>
   </property>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <property name="spacing">
      <number>1</number>
     </property>
     <item>
      <widget class="QToolButton" name="toolButtonExpandAll">
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset resource="../icon.qrc">
         <normaloff>:/DAGui/icon/expand-all.svg</normaloff>:/DAGui/icon/expand-all.svg</iconset>
       </property>
       <property name="autoRaise">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QToolButton" name="toolButtonCollapseAll">
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset resource="../icon.qrc">
         <normaloff>:/DAGui/icon/collapse.svg</normaloff>:/DAGui/icon/collapse.svg</iconset>
       </property>
       <property name="autoRaise">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="comboBoxFigure"/>
     </item>
     <item>
      <widget class="QToolButton" name="toolButtonFigureSetting">
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset resource="../icon.qrc">
         <normaloff>:/DAGui/icon/setting.svg</normaloff>:/DAGui/icon/setting.svg</iconset>
       </property>
       <property name="autoRaise">
        <bool>true</bool>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QStackedWidget" name="stackedWidget"/>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="../icon.qrc"/>
 </resources>
 <connections/>
</ui>
